### ISSUE TITLE
Screens with multiple pages - FR - have incorrect wording at the bottom  (iOS 1.0 (9))

### ISSUE THREAD

**Issue by mmiron-HC** [cds-snc/covid-shield-mobile#263]
in FR, just downloaded the new version of the app 1.0 (9)
Step 1 of 5 is listed at the bottom of the french screen.
![onboarding_step in EN](https://user-images.githubusercontent.com/65728700/85953575-6cf07e80-b93f-11ea-923a-98dbdfb52da5.jpg)

should be Étape 1 de 5
![image](https://user-images.githubusercontent.com/65728700/85953552-34e93b80-b93f-11ea-9aaa-b7b51a5338fb.png)

the issue is appearing on all screens that have multiple slides. (how covid alert works)

**Comment by JuliannaR** [comment]
This is fixed in V11. @mmiron-HC. Can you confirm?

**Comment by JuliannaR** [comment]
This is fixed in V11 and onward.
